### PR TITLE
Add guidelines on courses

### PR DESCRIPTION
**Pull request by FraGard**

### CODE REVIEW

File: _posts/2021-11-05-teaching-kit.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+categories: knowledge
+tags: teaching
+---
+
+## Developing a course from existing slides
+
+1. Prepare the general course information following these guidelines [course-info]
+2. Structure each lecture according to these guidelines [lecture-structure]
+3. Include images and media according to these guidelines [images]
+4. Publish the lecture addoring to these guidelines [publish-lecture]
```
Comment (willu47) [inline _posts/2021-11-05-teaching-kit.md:14]: Typo `adhering`

File: _posts/2021-11-05-teaching-kit.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Developing teaching material"
+date:   2021-11-05 14:30:00 +0200
```
Comment (willu47) [inline _posts/2021-11-05-teaching-kit.md:4]: The date of this post needs to be after the ones it links to.  Link to posts using the [post_url](https://jekyllrb.com/docs/liquid/tags/#linking-to-posts) tag.

**Review by willu47 — CHANGES_REQUESTED** [review]
Few minor changes - then good to go.

File: _posts/2021-11-05-teaching-kit.md (hunk 2 above)
Comment (FraGard) [reply _posts/2021-11-05-teaching-kit.md:4]: Correcting this. Confused about whether the url of the post should be the github url or the one of the published page. I suppose the latter?

Comment (willu47) [reply _posts/2021-11-05-teaching-kit.md:4]: Follow the link in the comment for post_url. That describes how to link to other posts using the name of the file. When the site is rendered, the markup will be replace by the correct url

**Review by willu47 — APPROVED** [review]

**Comment by willu47** [comment]
- I realised that you need to embed the `post_url variable` in a markdown link like so `[guidelines]({% post_url 2021-11-09-course-info %})` for it to render as a link.
- Removed headers which duplicated the title in the front_matter
    ```
    ---
    layout: post
    title:  "Structuring a lecture for new courses"
    date:   2021-11-17 10:30:00 +0200
    categories: knowledge
    tags: teaching
    ---
    ```

**Pull request merged**